bin/test.sh

```diff
@@ -111,6 +115,10 @@ if $ios82_only || $all_tests; then
     run_ios_tests "8.2" "ios82" "@skip-ios82|@skip-ios81|@skip-ios8|@skip-ios-all|@skip-ios7up"
 fi
 
+if $ios83_only || $all_tests; then
+    run_ios_tests "8.3" "ios83" "@skip-ios83|@skip-ios82|@skip-ios81|@skip-ios8|@skip-ios-all|@skip-ios7up"
```

Comment:
I've run them, and there are relatively few errors. I haven't had time to go through and run the failing tests individually to see if it they are just flakes. I also ran 8.2 and 8.1 tests on Xcode 6.3 beta 4, with similar results. I'll look into the details at some point.